cmd/manager/main.go

```diff
@@ -54,7 +54,10 @@ var (
 		prometheus.GaugeOpts{
 			Name:        "integreatly_version_info",
 			Help:        "Integreatly operator information",
-			ConstLabels: prometheus.Labels{"operator_version": version.Version},
+			ConstLabels: prometheus.Labels{
+				"operator_version": version.Version,
+				"integreatly_version" : version.IntegreatlyVersion,
```

Comment:
Can we use "version" as the label key on this metric.
This will make it easier for querying and visualising if all operators use the same metrics.

Reply:
Yes, will update

Reply:
@briangallagher just reopening this comment thread to ensure the key is changed to `version` instead of `integreatly_version`.
I think there might have been some confusion.

Here's an example from 3scale:
```
{
			Name: "threescale_version_info",
			Help: "3scale Operator version and product version",
			ConstLabels: prometheus.Labels{
				"operator_version": version.Version,
				"version":          product.ThreescaleRelease,
			},
```

and here's how i'd imagine it should look for integreatly:
```
{
			Name: "integreatly_version_info",
			Help: "Integreatly operator information",
			ConstLabels: prometheus.Labels{
				"operator_version": version.Version,
				"version":          version.IntegreatlyVersion,
			},
```